### ISSUE TITLE
Error linking as a library under Windows 

### ISSUE THREAD

**Issue by IanMichaelAsh** [abseil/abseil-cpp#702]
(Great project all, thank you!)

I have built a set of abseil-cpp libraries using Visual Studio / CMake under Windows, however when trying to link the library into a new project I am getting the following link error:

```
1>absl_random_internal_randen_hwaes_impl.lib(randen_hwaes.cc.obj) : error LNK2001: unresolved external symbol "unsigned char const * const absl::random_internal::kRandenRoundKeys" (?kRandenRoundKeys@random_internal@absl@@3QBEB)
1>absl_random_internal_randen_slow.lib(randen_slow.cc.obj) : error LNK2001: unresolved external symbol "unsigned char const * const absl::random_internal::kRandenRoundKeys" (?kRandenRoundKeys@random_internal@absl@@3QBEB)
```

Is there something in the make configuration which needs to be changed to build as a library? 

I'm a neophyte, so any insights on what I am missing would be greatly appreciated, as I'm stumped.

**Comment by derekmauro** [comment]
Please provide instructions on how to reproduce your problem. Thanks.

**Comment by IanMichaelAsh** [comment]
After doing a fresh clone of the abseil repo to generate a minimal example, the issue went away when I built a library from that fresh clone. 

It looks like the build process has changed since my original clone as the .lib file set is different (e.g. absl_atomic_hook_test_helper.lib doesn't exist) in the new clone vs my original.

Must have somehow ended up with an inconsistent build set after I tried to pull latest master into my clone a few days, leading to the issue I experienced.

Thanks.
